students/BrandonHenson/Lesson2/FizzBuzz.py

```diff
@@ -0,0 +1,25 @@
+'''
+Brandon Henson
+Python 210
+Lesson 2
+FizzBuzz
+4/1/18
+'''
+
+#For loop that iterates from 1 to 100 inclusively and
+#prints fizz,buzz,fizzbuzz or the number based on divisibilty.
+for i in range (1,101):
+#This if statement prints "FizzBuzz" if the number is divisible by both 4 AND 5
+        if (i % 3 == 0) and (i % 5 == 0):
```

Comment:
The parentheses are not necessary here, or in the `elif i % 3 == 0` statements below